### PR TITLE
feat(#5): Add `java` and `javac` methods to `Jhome`

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
Add `java` and `javac` methods to `Jhome`.

Closes: #5.

### CODE REVIEW

**Comment by yegor256** [comment]
@volodya-lombrozo in case of Java8 it's JRE, not JDK

**Comment by volodya-lombrozo** [comment]
@yegor256 Take a look one more time, please

File: src/main/java/com/yegor256/Jhome.java [hunk 1]
```diff
@@ -70,6 +70,26 @@ public Path path(final String loc) {
         return this.home.resolve(loc);
     }
 
+    /**
+     * Find the {@code java} binary.
+     * @return The path of it
+     */
+    public Path java() {
+        return this.home.resolve("bin").resolve("java");
```
Comment (yegor256) [inline src/main/java/com/yegor256/Jhome.java:78]: @volodya-lombrozo looks like code duplication to me. It's better to call `path("bin/java")` from here

File: src/main/java/com/yegor256/Jhome.java [hunk 2]
```diff
@@ -70,6 +70,26 @@ public Path path(final String loc) {
...
+    /**
+     * Find the {@code javac} binary.
+     * @return The path of it
+     * @todo #5 The javac binary might not be available in JRE.
+     *  We should check if we are running in JDK or JRE and somehow decide what to
+     *  return in this case or even throw an exception in case of JRE. Maybe an
+     *  {@code Optional<Path>} is also a solution here.
+     */
+    public Path javac() {
+        return this.home.resolve("bin").resolve("javac");
```
Comment (yegor256) [inline src/main/java/com/yegor256/Jhome.java:90]: @volodya-lombrozo same here, looks like code duplication

**Comment by yegor256** [comment]
@volodya-lombrozo I made it work on Windows (see the changes in the test). Would be great if in your pull request you also update README and explain that calling `.javac()` is better than using `"bin/java"` because of `.exe` file name extension that is present in Windows but is absent in Ubuntu.

**Comment by volodya-lombrozo** [comment]
@yegor256 Updated the PR according with your suggestions. Could you take a look one more time please?

**Comment by volodya-lombrozo** [comment]
@yegor256 I fix the problem with the Windows OS. Could you take a look, please?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**